.github/workflows/ci.yml

```diff
@@ -0,0 +1,64 @@
+name: CI
+on:
+  - push
+  - pull_request
+jobs:
+  test:
+    name: Julia ${{ matrix.version }} - ${{ matrix.os }} - ${{ matrix.arch }} - ${{ github.event_name }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        version:
+          - '1.0'
+          - '1' # automatically expands to the latest stable 1.x release of Julia
+          - 'nightly'
```

Comment:
This might be noisy as there isn't an equivalent to allow failures.

Reply:
We could have it in a separate step which is allowed to fail, like https://github.com/JuliaStats/StatsModels.jl/pull/199.

Reply:
FWIW the statsmodels setup doesn't actually allow it to fail in the strict sense that you'll still get the red X on the commit if it's failing.  but you can at least see isolate it in a separate workflow so you can have the nice green badge.

Reply:
Yeah, I guess if you aren't editing these files that often that's probably fine.